tests/pipeline/nodes/model/movenetv1/test_movenet.py

```diff
@@ -149,6 +137,26 @@ def test_no_human_single(self, empty_image, movenet_model_single):
                 ),
             )
 
+    def test_no_weights(self, movenet_config):
+        with mock.patch(
+            "peekingduck.weights_utils.checker.has_weights", return_value=False
+        ):
+            with mock.patch(
+                "peekingduck.weights_utils.downloader.download_weights",
+                wraps=replace_download_weights,
+            ):
+                with TestCase.assertLogs(
+                    "peekingduck.pipeline.nodes.model.movenetv1.movenet_model.logger"
+                ) as captured:
+                    movenet = Node(config=movenet_config)
+                    # records 0 - 20 records are updates to configs
+                    assert (
+                        captured.records[0].getMessage()
+                        == "---no weights detected. proceeding to download...---"
+                    )
+                    assert "weights downloaded" in captured.records[1].getMessage()
+                    assert movenet is not None
+
     def test_no_human_multi(self, empty_image, movenet_model_multi):
         no_human_img = cv2.imread(str(Path.joinpath(TEST_DIR, empty_image)))
         model, _ = movenet_model_multi
```

Comment:
`pytest -s tests/pipeline/nodes/model/movenetv1` on M1 MBP 13" completed successfully in 2m 33s.